Zend/zend_globals.h

```diff
@@ -127,6 +127,8 @@ struct _zend_compiler_globals {
 
 	HashTable *delayed_variance_obligations;
 	HashTable *delayed_autoloads;
+
+	size_t rtd_key_counter;
```

Comment:
why do  we need size_t? uint32_t should be more than enough.

Reply:
My original thought was that as this is a per-process/thread counter, a long living process may need more than 32-bits. However, as overflowing the counter is not a problem, I've changed this to uint32_t now.